tests/eclient/testdata/shutdown_test.txt

```diff
@@ -0,0 +1,73 @@
+# Test for shutdown of all app instances of EVE
+
+{{$port := "2223"}}
+{{$network_name := "n1"}}
+{{$app_name := "eclient"}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+[!exec:chmod] stop
+
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+# Create n1 network
+eden -t 1m network create 10.11.12.0/24 -n {{$network_name}}
+
+# Wait for run
+test eden.network.test -test.v -timewait 10m ACTIVATED {{$network_name}}
+
+eden pod deploy -n {{$app_name}} --memory=512MB docker://lfedge/eden-eclient:d9eb23f -p {{$port}}:22 --networks={{$network_name}}
+
+test eden.app.test -test.v -timewait 20m RUNNING {{$app_name}}
+
+# exec -t 5m bash ssh.sh
+# stdout 'Ubuntu'
+
+# send shutdown command
+eden controller edge-node shutdown
+
+# wait for HALTED state which indicates that we are shutting down
+test eden.app.test -test.v -timewait 5m HALTED {{$app_name}}
+# XXX can we check for the SHUTTING_DOWN state for the edge-node?
+
+# now reboot node to bring app back up
```

Comment:
Please sync timewaits inside this block. Timewait for app and network should be more than timewait for reboot.

Reply:
Updated